mollie/api/resources/base.py

```diff
@@ -36,37 +49,34 @@ def perform_api_call(self, http_method, path, data=None, params=None):
                 )
         return result
 
-    def _validate_resource_id(self, resource_id: str, name: str = "", message: str = ""):
-        resource_name = name or "Identifier"
-        message = (
-            message or f"Invalid {resource_name} '{resource_id}', it should start with '{self.RESOURCE_ID_PREFIX}'."
-        )
+    def validate_resource_id(self, resource_id: str, name: str = "Identifier", message: str = ""):
+        """Generic identifier validation."""
+        if not message:
+            message = f"Invalid {name} '{resource_id}', it should start with '{self.RESOURCE_ID_PREFIX}'."
 
         if not resource_id or not str(resource_id).startswith(self.RESOURCE_ID_PREFIX):
             raise IdentifierError(message)
 
     @staticmethod
     def extract_embed(params):
         """Extract and parse the embed parameter from the request."""
+        # TODO Remove this
```

Comment:
Is this TODO intentional?

Reply:
Yes, all the embed stuff is removed in commit c0803b693b93554a92664fddc0e87326018f248b